build.gradle.kts

```
@@ -33,8 +38,16 @@ dependencies {
     testImplementation("org.awaitility:awaitility:${property("awaitility.version")}")
 }
 
-tasks.getByName<Test>("test") {
+tasks.withType<Test>().configureEach {
     useJUnitPlatform()
+    maxHeapSize="6g"
```

just some formatting nitpicks
```suggestion
    maxHeapSize = "6g"
```